File: src/main/java/gq/noxiuam/bitches/object/type/EmoBitch.java
package gq.noxiuam.bitches.object.type;

import gq.noxiuam.bitches.object.Bitch;
import lombok.*;

/*
    Basically what 50% of Lunar Client's discord is
*/
@Getter @ToString
public class EmoBitch extends Bitch {

    private final boolean hasBarcodeWrists, wearsBlackClothes;

    public EmoBitch(boolean hasBarcodeWrists, boolean wearsBlackClothes) {
        this.hasBarcodeWrists = hasBarcodeWrists;
        this.wearsBlackClothes = wearsBlackClothes;
        this.build();
    }

}

File: src/main/kotlin/gq/noxiuam/bitches/object/type/EmoBitch.kt
package gq.noxiuam.bitches.`object`.type

import gq.noxiuam.bitches.`object`.Bitch
import lombok.Getter
import lombok.ToString

/*
    Basically what 50% of Lunar Client's discord is
*/
@Getter
@ToString
open class EmoBitch(private val hasBarcodeWrists: Boolean, private val wearsBlackClothes: Boolean) : Bitch() {
    init {
        this.build()
    }
}